idact/detail/cluster_app/main.py

```diff
@@ -0,0 +1,55 @@
+"""This module contains the :func:`main` function for adding new cluster
+ into local environment, see :mod:`idact.cluster`.
+
+ Note: The :func:`main` function uses :func:`click.command`, so it doesn't
+ show up in API docs for this module. See help message in :mod:`idact.cluster`
+ instead.
+
+"""
+from typing import Optional
+
+import click
+
+from idact import AuthMethod, KeyType
+
+@click.command()
+@click.argument('cluster_name',
+                type=str)
+@click.option('--user', '-u',
```

Comment:
Dlaczego optional? Optional chyba ma sens jeśli mamy jakąś defaultową wartość zmiennej, a tutaj nie ma. 
Widziałem, że w jupyter_app/main.py też jest optional i jednocześnie podany None jako default, ale później wartości defaultowe są pobierane z AppAllocationParameters

Reply:
Nie jestem pewna czy argumenty mogą mieć '--[coś]' przed sobą, a wydaje mi się że to polecenie jest na tyle długie że gdyby wpisywać wszystkie argumenty jeden za drugim to można by się było pogubić co jest czym.

Reply:
No ale skąd weźmiesz te argumenty jeśli ich nie wpiszesz skoro defaultowe _nie istnieją_?

Reply:
Cyba że @click.option nie implikuje że dany argument ma byc optionalem... Wtedy można po prostu w mainie usunąć Optionale i wyświetlać helpa jak nie będzie wszystkich danych. O to ci chodziło?